templates/views/notes.hbs

```diff
@@ -10,8 +10,28 @@
 
 <body>
     {{>navbar}}
-        
+        <section>
+            <div class="items">
+            <table class="table" border="0">
+                <thead class="tHead">
+                    <th scope="col">SNo</th>
+                    <th scope="col">Item Title</th>
+                    <th scope="col">Item Description</th>
+                    <th scope="col">Actions</th>
+                </thead>
+                <tbody id="tableBody" >
+                <tr>
+                    <th scope="row">1</th>
+                    <td>Coffee</td>
+                    <td>I love coffee</td>
+                    <td><button id="dBtn">Delete</button><button id="eBtn">Edit</button></td>
+                </tr>
+                </tbody>
+            </table>
+            </div>
+        </section>
     {{>footer}}
-</body>
+</body>    <script type="text/javascript"  src="./js/notes.js"></script>
```

Comment:
The new tag should be on a new line instead of in the same line. Move <script> tag to new line.